src/Apps/Artwork/Components/ArtworkDetails/index.tsx

```diff
@@ -19,10 +21,23 @@ export class ArtworkDetails extends Component<ArtworkDetailsProps> {
   render() {
     const { artwork } = this.props
     return (
-      <ArtworkDetailsContainer pb={3}>
-        <AboutTheWork artwork={artwork} />
-        <Checklist artwork={artwork} />
-        <AdditionalInfo artwork={artwork} />
+      <ArtworkDetailsContainer pb={4}>
+        <Tabs>
+          <Tab name="About the work">
+            <AboutTheWork artwork={artwork} />
+            <Checklist artwork={artwork} />
+            <AdditionalInfo artwork={artwork} />
+          </Tab>
+          <Tab name="Articles">
+            <Articles artwork={artwork} />
+          </Tab>
+          {artwork.exhibition_history && (
+            <Tab name="Exhibition history">{artwork.exhibition_history}</Tab>
+          )}
+          {artwork.literature && (
+            <Tab name="Bibliography">{artwork.literature}</Tab>
+          )}
+        </Tabs>
```

Comment:
@oxaudo - re

> I'd really like to pair for a bit with you tomorrow at some point @damassi to discuss this tabs here and if we need to do something more complicated here.

This working well for you? It seems exactly right

Reply:
The only question is finding a way to check that Articles is indeed renderable.

Reply:
Yes, the code above is indeed working for the last 2 tabs here. I am putting the last 2 tabs directly inline instead of creating a separate components for those though I think it's ok because both of those literally just displaying a text field per tab.

The first 2 tabs is where things get more complicated. And I think it makes sense to have components for each chunk of data separately but my Details parent component(index here) needs to know all the data to figure out what to render if anything at all. That is where things get tricky....

I tried to see what you guys did for Artist tabs and it seems like you and @mzikherman created the whole `ArtistStatuses` type in MP to hold the data for the tabs. You think it's a good approach or alternatively I could just query all data directly in `fragment ArtworkDetails_artwork on Artwork` instead of including child's fragments? All the data needs to be requested either way to know if tab should be rendered or not.

That is what I wanted to discuss:)
I might ask people around here for opinions as well meanwhile.

Reply:
@damassi - per our discussion in slack moved all the data getting into main ArtworkDetails component here. Kept all the rendering still split up into smaller components just to keep this file more concise and readable.

Let me know what you think here.

Reply:
Looks good!